modules/manage/pages/security/authorization/acl.adoc

```diff
@@ -1,14 +1,14 @@
-= Configure Authorization
+= Access Control Lists
 :description: ACLs are the main mechanism supported by Redpanda to manage user permissions.
-:page-aliases: features:acls.adoc, security:acls.adoc, security:authorization.adoc
+:page-aliases: features:acls.adoc, security:acls.adoc
 :page-categories: Management, Security
 
 
-Authorization works in tandem with xref:./authentication.adoc[authentication]. Access-control lists (ACLs) provide a way to configure fine-grained access to provisioned users. ACLs work with SASL/SCRAM and with mTLS with principal mapping for authentication.
+Access-control lists (ACLs) provide a way to configure fine-grained access to provisioned users. ACLs work with SASL/SCRAM and with mTLS with principal mapping for authentication.
 
 == ACLs
 
-Access-control lists (ACLs) are the primary mechanism used by Redpanda to manage user permissions. Redpanda stores ACLs internally, replicated with https://raft.github.io/[Raft^] to provide the same consensus guarantees as your data. You can manage ACLs with xref:reference:rpk/rpk-acl.adoc[`rpk acl`].
+Access-control lists (ACLs) are one of the primary mechanisms Redpanda provides to manage user permissions. Redpanda stores ACLs internally, replicated with https://raft.github.io/[Raft^] to provide the same consensus guarantees as your data. You can manage ACLs with xref:reference:rpk/rpk-acl.adoc[`rpk acl`].
```

Comment:
I'm torn here, because technically even RBAC falls back to ACLs to manage permissions, it just gives a container for grouping and assigning the ACLs. Are there other permissions management schemes in the works? Thoughts?